src/Mutant/Generator/MutationsGenerator.php

```diff
@@ -107,7 +107,7 @@ public function generate(bool $onlyCovered, string $filter = ''): array
         $this->eventDispatcher->dispatch(new MutationGeneratingStarted($files->count()));
 
         foreach ($files as $file) {
-            if (!$onlyCovered || ($onlyCovered && $this->hasTests($file))) {
+            if (!$onlyCovered || $this->hasTests($file)) {
                 $allFilesMutations[] = $this->getMutationsFromFile($file, $onlyCovered, $mutators);
             }
```

Comment:
Equivalent code. [See this discussion for details.](https://github.com/vimeo/psalm/issues/735)